### PR TITLE
[MAJOR] EDA-628 Token generator EDA-629 CompanyToken Model

### PR DESCRIPTION
**Pull request by ricardocaif-eb**
implemented token generator
implemented a companyunique token to keep track of company tokens and expiration

### CODE REVIEW

**Comment by lautaro-eb** [comment]
Can you provide a descriptive name to the commit?

File: supplier_app/tests/test_model.py [hunk 1]
```diff
@@ -134,3 +135,28 @@ def test_company_user_permissions(self):
...
+        self.assertEqual(
+            self.company,
+            self.companyuniquetoken.company
+        )
+
+    def test_company_token_does_not_repeat(self):
+        tokens = [
+            self.companyuniquetoken._token_generator() for i in range(100)
+        ]
+        self.assertTrue(len(tokens) == len(set(tokens)))
```
Comment (lautaro-eb) [inline supplier_app/tests/test_model.py]: I don't know if this is needed, it looks like you are testing the uuid library 🤔

File: supplier_app/tests/test_model.py [hunk 2]
```diff
@@ -134,3 +135,28 @@ def test_company_user_permissions(self):
...
+
+    def test_token_length(self):
+        token = self.companyuniquetoken._token_generator()
+        self.assertEqual(len(token), 64)
+
+    def test_company_unique_token_assing_company_token(self):
+        self.companyuniquetoken.assing_company_token
```
Comment (lautaro-eb) [inline supplier_app/tests/test_model.py]: No assertion here?

File: supplier_app/tests/test_model.py (hunk 1 above)
Comment (ricardocaif-eb) [reply supplier_app/tests/test_model.py]: Deleted!!

File: supplier_app/tests/test_model.py (hunk 2 above)
Comment (ricardocaif-eb) [reply supplier_app/tests/test_model.py]: Added assertion that a token has been assigned to companyuniquetoken

**Pull request merged**